.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,62 @@
+name: Python package
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+        python-version: [3.6, 3.7, 3.8]
```

Comment:
PRAW supports Python 3.5. Is there a way to test on that too?

Reply:
Black runs on python 3.6+ so that's why I don't lint on it but I do run unit tests on py3.5